.github/workflows/build.yml

```diff
@@ -48,6 +48,11 @@ jobs:
         echo "tests: false" >> cabal.project.local
         echo "benchmarks: false" >> cabal.project.local
 
+    - name: Disable -dynamic
```

Comment:
Why do we disable it here?

Reply:
Release builds are static, but we tell users to build dynamic. ..it's all quite confusing 😄

Afaiu static builds work fine as long as you use an official GHC bindist (does it work across ubuntu vs fedora bindists?)...otherwise TH may break.

Reply:
We want to build static binaries for ease of distribution. Sadly static binaries are no good for loading code dynamically, which is needed for compiling TH code.

Reply:
> Afaiu static builds work fine as long as you use an official GHC bindist (does it work across ubuntu vs fedora bindists?)...otherwise TH may break.

Yeah i guess those are the users who had th working but have to close the warning or build hls from source without needing it.